routes/index.js

```diff
@@ -29,7 +30,29 @@ app.get("/clients_weights/:clientId", db.getWeights);
 
 app.post("/clients_weights/:clientId", db.addWeight);
 
-app.post("/clients", db.addClient);
+app.post("/clients", (req, res, next) => {
+  //req.body.client_name = client name
+  if (!req.body.client_name) {
+    res.status(400).send("Client name required.");
+  }
+  if (!req.body.birth_day) {
+    res.status(400).send("Birthday required.");
+  }
+  if (Date.parse(req.body.birth_day) < -2208967764000) {
+    console.log(Date.parse(req.body.birth_day));
+    res.status(400).send("Are you really that old?");
+  }
```

Comment:
Let's try making this dynamic. How could you calculate a certain amount of years (120?) in the past? So say this app is running for 10 years, and you'd never have to change the code but it would always validate that a person can't be more than 120 years old.

Reply:
made it dynamic